pyscitt/pyscitt/prefix_tree.py

```diff
@@ -110,8 +110,9 @@ def verify(
         self, claim: Union[Sign1Message, bytes], service_params: ServiceParameters
     ):
         if isinstance(claim, bytes):
-            claim = CoseMessage.decode(claim)
-        assert isinstance(claim, Sign1Message)
+            claim_ = CoseMessage.decode(claim)
+            assert isinstance(claim_, Sign1Message)
+            claim = claim_
```

Comment:
`Sign1Message.decode(claim)` should have the right type, thanks to https://github.com/TimothyClaeys/pycose/pull/102, which means you shouldn't need the `assert` and temporary variable.